### DISCUSSION TITLE
Final merger

### DISCUSSION THREAD

**Discussion by 000961892** [MarkW06071969/Tafe202206#21] [General]
Do you guys want to meet up in A.114 tomorrow at 1:00pm to perform the final merger

**Comment by MarkW06071969** [comment]
Hi Hamish,
That would be great.
See you then
Sent from Yahoo Mail on Android 
 
  On Wed, 7 Sept 2022 at 8:30 am, ***@***.***> wrote:   


Do you guys want to meet up in A.114 tomorrow at 1:00pm to perform the final merger

**Reply by StephenHillenbrand** [reply]
Can do. See you then.

**Reply by StephenHillenbrand** [reply]
Might be an idea to read up on pull requests as we need to fork the main branch after and then perform a pull request. A few other groups are having trouble with this. 

KT has a recorded session on this. 

Be good to get this completed as well tomorrow.

**Comment by MarkW06071969** [comment]
Good idea

Sent from Yahoo Mail on Android 
 
  On Wed, 7 Sept 2022 at 8:51 am, ***@***.***> wrote:   


Might be an idea to read up on pull requests as we need to fork the main branch after and then perform a pull request. A few other groups are having trouble with this.

KT has a recorded session on this.

Be good to get this completed as well tomorrow.
